### ISSUE TITLE
Running python run_alphafold_test. py encountered ImportError: /lib64/libc.so.6: version `GLIBC_2.28' not found

### ISSUE THREAD

**Issue by Kangfengyuuuu** [google-deepmind/alphafold3#126]
Hello everyone, I have been using online tutorials to install AlphaFold 3 using Conda, and then running `python run_alphafold_test.py`. I encountered the problem：

```
ImportError: /lib64/libc.so.6: version `GLIBC_2.28' not found (required by /home/kangfengyu/miniconda3/envs/alphafold3/lib/python3.11/site-packages/rdkit/../rdkit.libs/libboost_filesystem-47d2d140.so.1.85.0)
```

After checking many tutorials, I was afraid that the Linux would crash. Now I don't know how to solve it. Has anyone encountered the same problem?

**Comment by Augustin-Zidek** [comment]
This looks like a problem with your Conda setup (rather than anything AlphaFold-specific), take a look at https://github.com/rdkit/rdkit/issues/4756.

**Comment by vivek-rd** [comment]
I faced a similar issue (GLIBC_2.xx version not found) in a different project. If you installed the dependencies in an existing conda environment, try installing the dependencies in a new conda environment. This helped me resolve it.

**Comment by Kangfengyuuuu** [comment]
> 我在另一个项目中遇到了类似的问题（未找到 GLIBC_2.xx 版本）。如果您在现有 conda 环境中安装了依赖项，请尝试在新的 conda 环境中安装依赖项。这帮助我解决了这个问题。

I now know that the program called libc.so.6 in Linux, which is incorrect. It should have called the dynamic library in conda, but I couldn't find libc.so.6 in my conda environment, and I downloaded GCC in a new conda environment, but I couldn't find libc.so.6 either. May I ask how you specifically solved it? There are many solutions online about libstd c++. so. 6, but I haven't found any specific way to solve libc. so. 6.

**Comment by Kangfengyuuuu** [comment]
> This looks like a problem with your Conda setup (rather than anything AlphaFold-specific), take a look at [rdkit/rdkit#4756](https://github.com/rdkit/rdkit/issues/4756).

I couldn't find a solution to libc. so. 6, so I tried installing Rdkit=2024.3.2 and ran run_alphafold_test. py without any more errors. 
Does installing Rdkit=2024.3.2 have a significant impact on the prediction results of Alphafold 3?

**Comment by vivek-rd** [comment]
> > 我在另一个项目中遇到了类似的问题（未找到 GLIBC_2.xx 版本）。如果您在现有 conda 环境中安装了依赖项，请尝试在新的 conda 环境中安装依赖项。这帮助我解决了这个问题。
> 
> I now know that the program called libc.so.6 in Linux, which is incorrect. It should have called the dynamic library in conda, but I couldn't find libc.so.6 in my conda environment, and I downloaded GCC in a new conda environment, but I couldn't find libc.so.6 either. May I ask how you specifically solved it? There are many solutions online about libstd c++. so. 6, but I haven't found any specific way to solve libc. so. 6.

In my case, I started with a completely new conda environment. I did not install libc directly in the new environment and I just installed all the required dependencies and it resolved this issue.

**Comment by Kangfengyuuuu** [comment]
> > > 我在另一个项目中遇到了类似的问题（未找到 GLIBC_2.xx 版本）。如果您在现有的 conda 环境中安装了依赖项，请尝试在新的 conda 环境中安装依赖项。这帮助我解决了这个问题。
> > 
> > 
> > 我现在知道程序在linux下调用的是libc.so.6，这是不对的，应该是在conda中调用动态库，但是我在我的conda环境中找不到libc.so.6，我在新的conda环境中下载了GCC，也找不到libc.so.6，请问您具体是怎么解决的？网上有很多关于libstd c++.so.6的解决方案，但是我没有找到解决libc.so.6的具体方法。
> 
> 就我而言，我从一个全新的 conda 环境开始。我没有在新环境中直接安装 libc，而是安装了所有必需的依赖项，这样就解决了这个问题。
Thank you for your answer! I solved the error of GLIBC-2.28 'not found' by reducing the version of rdkit. I will try reinstalling all dependencies in the new conda environment again.

**Comment by JeffersonPark1304** [comment]
@Kangfengyuuuu I faced the same issue.  The following step solved my issue. Hopefully, it will help you

conda install libgcc
